.github/workflows/build-test.yml

```diff
@@ -42,6 +42,31 @@ jobs:
       - name: Check license header
         run: ./internal/scripts/ci_test_spdx_license_header.sh
 
+      - name: Setup GitHub CLI
```

Comment:
In this file, line 28 I think you find a filter that all code-related tests are skipped when only `md` files have changed. (I am a bit unsure if it is just this line but we added it to safe some CI time).

So could you add a filter that the markdown checks are only executed when `md` files have been changed or it is always executed - otherwise we may introduce some markdown warnings with the previously mentioned filter when only `md` files have been changed and then everything is skipped.

Reply:
@elfenpiff Added: https://github.com/eclipse-iceoryx/iceoryx2/pull/387/commits/45de7464d4bdb6aa8ada7e0c19a5246d525f4170